File: shriyashahhw.py
import json

# Sample JSON string (you can replace it with the actual JSON data from a file)
json_data = '''
{
    "name": "shriya",
    "age": 16,
    "is_student": true,
    "skills": ["Python", "JavaScript", "SQL"],
    "address": {
        "city": "San Diego",
        "state": "California"
    }
}
'''

# Load JSON data
data = json.loads(json_data)

# Edit the JSON data with different types of variables
data['name'] = "Swati Patel"          # String
data['age'] = 45                   # Integer
data['is_student'] = False         # Boolean
data['skills'].append("C++")       # List
data['address']['zip_code'] = 92127 # Dictionary (Adding new key-value pair)

# Print updated JSON data
print(json.dumps(data, indent=4))

File: shriyashahhw.js
// Sample JSON data (you can replace it with actual JSON)
let jsonData = {
    "name": "Shriya",
    "age": 16,
    "is_student": true,
    "skills": ["Python", "JavaScript", "SQL"],
    "address": {
        "city": "San Diego",
        "state": "California"
    }
};

// Edit the JSON data with different types of variables
jsonData.name = "Swati Patel";          // String
jsonData.age = 45;                   // Integer
jsonData.is_student = false;         // Boolean
jsonData.skills.push("C++");         // Array
jsonData.address.zip_code = 94105;   // Object (Adding new key-value pair)

// Log the updated JSON data
console.log(JSON.stringify(jsonData, null, 4));